recipes/hidapi/all/conanfile.py

```diff
@@ -0,0 +1,93 @@
+# Known Issues
+#
+# * Not support x64 for Windows
+#
+#     [libusb/hidapi](https://github.com/libusb/hidapi) provides a `sln` file to build for Windows, which lack support x64.
+#
+# If you get an idea to solve one of this issues, please report here or fork.
+import os
+from conans import ConanFile, AutoToolsBuildEnvironment, MSBuild, tools
+
+
+class HidapiConan(ConanFile):
+    name = "hidapi"
+    description = "HIDAPI is a multi-platform library which allows an application to interface with USB and Bluetooth HID-Class devices on Windows, Linux, FreeBSD, and macOS."
+    topics = ("conan", "hidapi", "libusb")
+    url = "https://github.com/conan-io/conan-center-index"
+    homepage = "https://github.com/libusb/hidapi"
+    license = "BSD-Style"
+    settings = "os", "compiler", "build_type", "arch"
+    options = {
+        "minosx": ['10.7', '10.8', '10.9', '10.10', '10.11', '10.12', '10.13', '10.14', '10.15', '11'],
+        "fPIC": [True, False],
+        "with_libusb": [True, False]
+    }
+    default_options = {
+        "minosx": 10.7, "fPIC": True, "with_libusb": False
+    }
+    generators = "pkg_config"
+
+    @property
+    def _source_dir(self):
+        return "hidapi-hidapi-" + self.version
+
+    def config_options(self):
+        if self.settings.os != "Macos":
+            del self.options.minosx
+        if self.settings.os != "Linux":
+            del self.options.with_libusb
+        if self.settings.os == "Windows":
+            self.settings.arch.remove("x86_64")
+
+    def requirements(self):
+        if self.settings.os == "Linux" or self.settings.os == "FreeBSD":
+            self.requires("libusb/[~=1.0]")
```

Comment:
Do not use version ranges, use the exact requirement version. The underlying reason for this is that different requirement versions might result in different packageID and we want users consuming ConanCenter to have binaries available when they retrieve a package.